octokit/pull_requests.go

```diff
@@ -91,6 +91,7 @@ type PullRequest struct {
 	Additions         int               `json:"additions,omitempty"`
 	Deletions         int               `json:"deletions,omitempty"`
 	ChangedFiles      int               `json:"changed_files,omitempty"`
+	Mergeable         *bool             `json:"mergeable,omitempty"`
```

Comment:
There's also `mergeable_state` which returns one of "clean", "unstable", "dirty", or "unknown". Interested in that one?